feat:키보드 레이아웃 마무리

leehandsub opened the pull request:
책 search부분관 메인 search부분 엔터 입력시 동작되게 만들었습니다.
post,modify부분 제목 가격 엔터입력시 다음으로 넘어가게했습니다.
post,modify 내용 입력시는 원하는대로 되지 않지만 입력은 잘되게 햿습니다.

leehandsub committed:
feat:키보드 레이아웃 마무리

책 search부분관 메인 search부분 엔터 입력시 동작되게 만들었습니다.
post,modify부분 제목 가격 엔터입력시 다음으로 넘어가게했습니다.
post,modify 내용 입력시는 원하는대로 되지 않지만 입력은 잘되게 햿습니다.

YeseopLee commented:
확인했습니다.

1. 가격에서 엔터 입력시 지역 다이얼로그가 뜨게끔 부탁드립니다
2. 여전히 검색(메인화면 통합검색 및 네이버북 api 검색)에서 엔터 입력시 바로 검색이 안되고 개행이 되는것 같습니다.

leehandsub committed:
Merge branch 'develop' of https://github.com/leehandsub/bookwhale-client into keyboard

leehandsub committed:
feat: post,modify, search, main 키보드 동작 구현

modify search 가격다음 엔터누르면 위치 다이얼로그 실행
search 검색이 실행되면 키보드 없어지기
main 검색창이 없을때 돋보기 버튼 누를 시 키보드 올라가고
검색창이 있을때 돋보기나 엔터를 누를 시 키보드가 내려갑니다.

문제 : 처음 앱이 실행되고 찾기 버튼 누를시 키보드가 안올라오네요...

YeseopLee commented:
확인했습니다!
같은 작업을 채팅방이랑 내정보 화면에도 적용시켜주시면 될 것 같아요.

*처음 앱이 실행되고 찾기 버튼 누를시 키보드가 안올라오네요...

이부분은 아마 focusable 관련해서 처리하면 될 것 같은데.. 한번 그쪽으로 찾아보시면 좋을것같아요.

leehandsub committed:
Merge branch 'develop' of https://github.com/leehandsub/bookwhale-client into keyboard

leehandsub committed:
feat : 내정보 수정

leehandsub committed:
Merge branch 'develop' of https://github.com/leehandsub/bookwhale-client into keyboard

leehandsub committed:
feat : 맨 처음 키보드 올리기

leehandsub committed:
feat : 내 정보 완료후 키보드 내리기

leehandsub commented:
내 정보 관련 된 키보드 설정 완료
메인에서 맨 처음 돋보기 버튼 누르면 키보드 올라오게 했습니다.

YeseopLee commented:
확인하였습니다.
